2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Configure stats pid to 26360
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Loading settings from C:\Users\zaynm\.config\wandb\settings
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Loading settings from d:\AiProjects\PneumoniaClassifier\wandb\settings
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:_log_setup():534] Logging user logs to d:\AiProjects\PneumoniaClassifier\wandb\run-20241104_170041-4zgf6xgn\logs\debug.log
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:_log_setup():535] Logging internal logs to d:\AiProjects\PneumoniaClassifier\wandb\run-20241104_170041-4zgf6xgn\logs\debug-internal.log
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:_jupyter_setup():480] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000022C072BF6B0>
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:init():621] calling init triggers
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:init():671] starting backend
2024-11-04 17:00:41,144 INFO    MainThread:26360 [wandb_init.py:init():675] sending inform_init request
2024-11-04 17:00:41,145 INFO    MainThread:26360 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2024-11-04 17:00:41,146 INFO    MainThread:26360 [wandb_init.py:init():688] backend started and connected
2024-11-04 17:00:41,148 INFO    MainThread:26360 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-11-04 17:00:41,149 INFO    MainThread:26360 [wandb_run.py:_label_probe_notebook():1352] Unable to probe notebook: 'charmap' codec can't decode byte 0x8f in position 364017: character maps to <undefined>
2024-11-04 17:00:41,149 INFO    MainThread:26360 [wandb_init.py:init():783] updated telemetry
2024-11-04 17:00:41,176 INFO    MainThread:26360 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-11-04 17:00:41,446 INFO    MainThread:26360 [wandb_init.py:init():867] starting run threads in backend
2024-11-04 17:00:41,507 INFO    MainThread:26360 [wandb_run.py:_console_start():2463] atexit reg
2024-11-04 17:00:41,507 INFO    MainThread:26360 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-11-04 17:00:41,507 INFO    MainThread:26360 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-11-04 17:00:41,507 INFO    MainThread:26360 [wandb_run.py:_redirect():2401] Redirects installed.
2024-11-04 17:00:41,507 INFO    MainThread:26360 [wandb_init.py:init():911] run started, returning control to user process
2024-11-04 17:00:41,508 INFO    MainThread:26360 [wandb_watch.py:_watch():71] Watching
2024-11-04 17:18:18,471 INFO    MainThread:26360 [wandb_run.py:_finish():2158] finishing run zheng-yu3-ucl/chest_xray_classification/4zgf6xgn
2024-11-04 17:18:18,472 ERROR   MainThread:26360 [jupyter.py:save_history():451] Run pip install nbformat to save notebook history
2024-11-04 17:18:18,472 INFO    MainThread:26360 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-11-04 17:18:18,472 INFO    MainThread:26360 [wandb_init.py:_jupyter_teardown():462] cleaning up jupyter logic
2024-11-04 17:18:18,472 INFO    MainThread:26360 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-11-04 17:18:18,472 INFO    MainThread:26360 [wandb_run.py:_restore():2408] restore
2024-11-04 17:18:18,472 INFO    MainThread:26360 [wandb_run.py:_restore():2414] restore done
2024-11-04 17:18:21,487 INFO    MainThread:26360 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-11-04 17:18:21,488 INFO    MainThread:26360 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-11-04 17:18:21,506 INFO    MainThread:26360 [wandb_run.py:_footer_sync_info():3934] logging synced files
